amy/templates/requests/all_trainingrequests.html

```diff
@@ -62,6 +64,13 @@
             &lt;{{ req.email|urlize }}&gt;
           </td>
           <td>{{ req.member_code|default:"—" }}</td>
+          <td>
+            {% if req.eventbrite_url %}
+              {% eventbrite_id_from_url req.eventbrite_url %}
+            {% else %}
+              —
```

Comment:
Can you use `&mdash;` entity for a default value here?

Edit: For context, I think I introduced the default value of `&mdash;`, but then changed my mind and started using `—` as I was able to type it from keyboard. Now I'm not able to type it easily, so I prefer the original approach of `&mdash;` (it's also less confusing).